Tasks/Integration/02_ManageAttributeSetsTasks.md

```diff
@@ -0,0 +1,19 @@
+[!!User interface Attribute sets](../UserInterface/02b_AttributeSets.md)
+[!!Manage an attribute set in DataHub](../../DataHub/Integration/02_ManageAttributeSets.md)
+
+# Manage an attribute set for tasks
+
+An attribute set consist of multiple attributes to classify certain entity type, in this case, a task. In the *Tasks* module, you can create, edit, and deactivate attribute sets that are specific to your tasks. 
+
+The *ATTRIBUTE SETS* tab in the *Tasks* module equals the *ATTRIBUTE SETS* tab contained in the *DataHub* module. In the *Tasks* module, however, only the tasks-relevant attribute sets are displayed. 
+
+For detailed information on how to create, edit, and deactivate attribute sets, see [Manage an attribute](../../DataHub/Integration/02_ManageAttributeSets.md) in the *DataHub* documentation.
+
+[comment]: <> (Bitte LOCK-Button aus Editing Toolbar abbauen!)
+
+[comment]: <> (so allgemein, oder unter dem jeweiligen Procedure?)
```

Comment:
ich finde, der allgemeine Link reicht. Ich würde auch nur schreiben: For detailed information on how to manage an attribute, see [Manage an attribute]